chromium_src/components/search_engines/brave_template_url_service_util_unittest.cc

```diff
@@ -45,15 +46,47 @@ class BraveTemplateURLServiceUtilTest : public testing::Test {
   sync_preferences::TestingPrefServiceSyncable prefs_;
 };
 
-TEST_F(BraveTemplateURLServiceUtilTest, GetSearchProvidersUsingKeywordResult) {
+void TestDefaultOrder(const TemplateURL::OwnedTemplateURLVector& template_urls,
+                      std::vector<std::string> keywords) {
```

Comment:
Any reason not to pass the last param as `const std::vector<std::string>& keywords`?

Reply:
Fixed with https://github.com/brave/brave-core/pull/3591/commits/201436c15a1660d11fc108ac1495933e2f8358ea 👍